P/Pango/build_tarballs.jl

```diff
@@ -68,9 +68,9 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Cairo_jll"; compat="1.16.1"),
+    Dependency("Cairo_jll"; compat="1.18.0"),
     Dependency("Fontconfig_jll"),
-    Dependency("FreeType2_jll"; compat="2.10.4"),
+    Dependency("FreeType2_jll"; compat="2.13.1"),
```

Comment:
For the record, this is because #7454 changed the compat bound of `FreeType2_jll` to 2.13.1, but I have no clue of why: https://github.com/JuliaPackaging/Yggdrasil/pull/7454/files#diff-e2d8f4bfb63a6c0200a9e9beeb578adef317bf2a540738ea6b7858df24e1115fR76